.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+# Create a new GitHub release on pushed tag stating with 'v'.
+# Run tests, build and uploads binaries.
+
+name: Release
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+
+  test:
+    name: Release Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+
+    - name: sanity-check
+      run: make test
+
+  # See https://github.com/actions/upload-release-asset
+  build:
+    name: Build and Release
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+
+    - name: build Linux
+      run: |
+        make build-linux
+        zip --junk-paths terraform-provider-rollbar-linux terraform-provider-rollbar
+        rm terraform-provider-rollbar
+
+    - name: build Darwin
+      run: |
+        make build-darwin
+        zip --junk-paths terraform-provider-rollbar-darwin terraform-provider-rollbar
+        rm terraform-provider-rollbar
+
+    - name: create release
+      id: create-release 
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: false
+        prerelease: true
+
+    - name: upload Release Asset Linux
+      id: upload-release-asset-linux
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create-release.outputs.upload_url }}
+        asset_path: ./terraform-provider-rollbar-linux.zip
+        asset_name: terraform-provider-rollbar-linux.zip
```

Comment:
I think the name is different now than the one uploaded on travis-ci. The old one also included the architecture `amd64`. This was meant to be similar to the official setup and is also important when downloading it